### PR TITLE
feat: new styles and rss feed

### PR DESCRIPTION
**Pull request by EzequielCaste**
Hey, that's a big commit. 🥇 

I changed some styles. Played around with remark and react-remark.
Got the RSS feed working.
Added the first blog post.

modified:   _posts/hello world.md
modified:   components/hero-post.js
modified:   components/layout.js
modified:   components/more-stories.js
modified:   components/navbar.js
modified:   lib/constants.js
modified:   lib/markdownToHtml.js
new file:   lib/rss.js
new file:   next.config.js
modified:   package-lock.json
modified:   package.json
modified:   pages/blog.js
modified:   pages/posts/[slug].js
new file:   public/images/c.gif
new file:   public/images/duck.png
new file:   public/images/python-welcome.jpg
new file:   public/images/visual-basic.gif
new file:   public/rss.xml
modified:   styles/globals.css
modified:   tailwind.config.js

### CODE REVIEW

File: components/layout.js [hunk 1]
```diff
@@ -24,18 +24,21 @@ export default function Layout({ preview, children }) {
...
         <meta name="msapplication-TileColor" content="#000000" />
         <meta
           name="msapplication-config"
           content="/favicon/browserconfig.xml"
         />
         <meta name="theme-color" content="#000" />
+
+        <link rel="preconnect" href="https://fonts.gstatic.com" />
+        <link
+          href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap"
```
Comment (mobregozo) [inline components/layout.js]: just in case! please do check if we are using all the weights

**Review by mobregozo — APPROVED** [review]

File: components/layout.js (hunk 1 above)
Comment (EzequielCaste) [reply components/layout.js]: yeah, I just checked and we're only using medium and bold (500,700).

I can fix that right now 🚧

Comment (EzequielCaste) [reply components/layout.js]: nope... actually it's 400,500,600 and 700 👓

**Review by mobregozo — COMMENTED** [review]
looks good to me!

**Pull request merged**